app/resources/views/pages/valhalla/round-type.blade.php

```diff
@@ -35,7 +35,11 @@
                         <tr>
                             @foreach ($row as $column => $value)
                                 <td{!! (isset($headers[$column]['align-center']) && $headers[$column]['align-center']) ? ' class="text-center"' : null !!}>
-                                    {{ $value }}
+                                    @if ($column == 'player')
+                                        {!! $value !!}
```

Comment:
Why was this added?

Reply:
I added a link to the 'lifetime' page which shows land/networth for all of that users' rounds played. Since the template is the same for all of the Valhalla pages and the displayed data has to be generated in a map function in the Controller, the link is generated added there and has to be escaped to appear on the page. I only did it for the player column just to prevent any unintended side-effects.

Reply:
I'm slightly concerned some players try be funny and put HTML in their dom name, which would normally be 'fine' (since it gets escaped everywhere), but then except here. Would this be a risk? I think you have a better grasp of the context regarding this than I do atm 😋

Reply:
Looks like I tested that. It escapes (somewhat) the player name within the link that gets generated. Not sure if that's 100% safe, but it worked for simple html/javascript stuff I tried.

_htmlentities($dominion->user->display_name)_